module_renamer/cli.py

```diff
@@ -1,16 +1,34 @@
 # -*- coding: utf-8 -*-
-
 """Console script for module_renamer."""
+from __future__ import absolute_import
+
 import sys
+
 import click
 
+from .commands.track_modifications import track_modifications
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+def main():
+    """
+    Console script for module_renamer.
+    """
+    pass
+
 
-@click.command()
-def main(args=None):
-    """Console script for module_renamer."""
-    click.echo("Replace this message by putting your code into "
-               "module_renamer.cli.main")
-    return 0
+@main.command()
+@click.argument('project_path', type=click.Path(exists=True))
+@click.option('--origin_branch', default='master',
```

Comment:
I was not using the dashes because I was not being able to access it, but I just found that any internal " - " characters will be converted to underscore.
The next commit improves this inputs ;)